### PR TITLE
[cmake] Support Ubuntu 20.04

### PR DESCRIPTION
**Pull request by deniskoronchik**
* [ ] Read PR [documentation](http://ostis-dev.github.io/sc-machine/dev/pr/)
* [ ] Update changelog
* [ ] Update documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,43 @@
...
+pip3 install -r requirements.txt
+```
+
+__Ubuntu 18.04__ and __Ubuntu 20.04__
+
+```sh
+sudo apt install librocksdb-dev libglib2.0-dev qtbase5-dev libboost-system-dev libboost-filesystem-dev libboost-program-options-dev make cmake antlr gcc g++ llvm-7 libcurl4-openssl-dev libclang-7-dev libboost-python-dev python3-pip
+```
+
+Install Python evelopment package
```
Comment (ValikVityaz) [inline README.md:22]:
```suggestion
Install Python development package
```

File: README.md [hunk 2]
```diff
@@ -1,20 +1,43 @@
...
-**Master:** 
-[![Build Status](https://travis-ci.com/ostis-dev/sc-machine.svg?branch=master)](https://travis-ci.com/ostis-dev/sc-machine)
+## Build on Ubuntu
+
+### Dependencies
+
+Install python dependencies:
+
+```sh
+pip3 install -r requirements.txt
```
Comment (ValikVityaz) [inline README.md:13]: Can fail if `pip3 install setuptools wheel` hasn't been run in advance.

**Review by ValikVityaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (deniskoronchik) [reply README.md:13]: Never got problems with such way. Can you share any documentation on this?

Comment (ValikVityaz) [reply README.md:13]: It's likely 'cause setuptools & wheel were already installed on your system.
https://packaging.python.org/tutorials/installing-packages/#ensure-pip-setuptools-and-wheel-are-up-to-date
I've tried to run `pip3 install -r requirements.txt` on a freshly installed Linux Mint 20 and failed due to the absence of these packages.

Comment (deniskoronchik) [reply README.md:13]: This is a Mint specific problem, can you update instruction in `readme` to support Mint?

**Review by ValikVityaz — APPROVED** [review]

**Review by KovalM — APPROVED** [review]

**Pull request merged**